build.psm1

```diff
@@ -220,6 +220,10 @@ function Start-PSBuild {
         [switch]$NoPSModuleRestore,
```

Comment:
@rjmholt, your last commit had 10 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

CimInstance cmdlet tests.CimClass property should not be null
```powershell
Expected regular expression '_computersystem' to match $null, but it did not match.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/CimCmdlets/CimInstance.Tests.ps1: line 18
18:             $instance.CimClass.CimClassName | Should -Match _computersystem
```
CimInstance cmdlet tests.GetCimSessionInstanceId method invocation should return data
```powershell
You cannot call a method on a null-valued expression.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/CimCmdlets/CimInstance.Tests.ps1: line 32
32:             $instance.GetCimSessionInstanceId() | Should -BeOfType "Guid"

```
CimInstance cmdlet tests.should produce an error for a non-existing classname
```powershell
Expected an exception, with FullyQualifiedErrorId 'HRESULT 0x80041010,Microsoft.Management.Infrastructure.CimCmdlets.GetCimInstanceCommand' to be thrown, but the FullyQualifiedErrorId was 'CommandNotFoundException'. from /home/vsts/work/1/s/test/powershell/Modules/CimCmdlets/CimInstance.Tests.ps1:36 char:15
    +             { Get-CimInstance -ClassName thisnameshouldnotexist -Erro ?
    +               ~~~~~~~~~~~~~~~
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/CimCmdlets/CimInstance.Tests.ps1: line 37
37:                 Should -Throw -ErrorId "HRESULT 0x80041010,Microsoft.Management.Infrastructure.CimCmdlets.GetCimInstanceCommand"
```
New-CimSession.A cim session can be created
```powershell
The term 'Remove-CimSession' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/CimCmdlets/CimSession.Tests.ps1: line 14
14:                 $sessions | Remove-CimSession -ErrorAction SilentlyContinue

```
New-CimSession.A Cim session can be retrieved
```powershell
The term 'Remove-CimSession' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/CimCmdlets/CimSession.Tests.ps1: line 14
14:                 $sessions | Remove-CimSession -ErrorAction SilentlyContinue

```